docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -924,17 +924,166 @@ intended distribution.
+    - 4.34
+    - 4.1
+  * - Hispanic/Latino 
+    - 4.99
+    - 4.75 
+
+
+.. list-table:: Simulant Omission by Age/Sex 
+  :widths: 20 10 10 
+  :header-rows: 1
+
+  * - Age/Sex
+    - Percent Omitted 
+    - Additive Risk of Omission (%)
+  * - US Total (all ages and sexes) 
+    - 0.24
+    - 0 (reference)
+  * - 0-4, all sexes 
+    - 2.79
+    - 2.55
+  * - 5-9, all sexes 
+    - 0.1
+    - -0.14
+  * - 10-17, all sexes 
+    - 0.21
+    - -0.03
+  * - 18-29, male 
+    - 2.25
+    - 2.01
+  * - 18-29, female 
+    - 0.98
+    - 0.74
+  * - 30-49, male 
+    - 3.05
+    - 2.81
+  * - 30-49, female 
+    - -0.1
+    - -0.34
+  * - 50+, male 
+    - -0.55
+    - -0.79
+  * - 50+, female 
+    - -2.63
+    - -2.87
+
+Using the tables above, a probability of omission is calculated for 
+each simulant. The table below includes a few examples of this 
+process. 
+
+.. list-table:: Calculating Simulant Omission 
+  :widths: 10 10 10 20
+  :header-rows: 1
+
+  * - Simulant 
+    - Race/Ethnicity 
+    - Age/Sex
+    - Probability Risk of Omission (%)
+  * - 1
+    - White 
+    - 0-4, female 
+    - 0.24 + (-1.88) + (2.55) = **0.91%** 
+  * - 2
+    - Black 
+    - 30-49, male 
+    - 0.24 + (3.06) + (2.81) = **6.11%** 
+  * - 3
+    - Asian 
+    - 50+, female 
+    - 0.24 + (-2.86) + (-2.87) = -5.49 is < 0 so **0%** 
+
+Please note that for simulants with a net undercount less than 0, 
+they have a 0% chance of being missed. We are not including 
+duplicates at this time. 
+
+
+**Data Errors/Noise** 
+In the future, we will add a noise function decided to replicate 
+missed or incorrect data. This includes incorrect name spelling, 
+addresses, age or DOB, and person swaps. 
+
+Some errors will be introduced within the simulation, such as 
+duplicate people or swapped people. Others will be introduced 
+at the time of sampling, such as name and address misspellings. 
+
+These are not currently modeled. 
+
+**Limitations and Possible Future Adds** 
+
+#. Sampling on a single time step is not representative of the true census. People might move houses, change names, have babies, or have loved ones die during the census leading to additional noise in the census not modeled here 
+#. Our model will underestimate total census coverage as we are not including net overcounts for certain population segments 
+#. Here we model a net undercount rather than modeling duplications and omissions separately. In reality, simulants are both duplicated and omitted within each race/age/sex group which leads to additional noise in the data 
+#. There are multiple other factors that contribute to omission rate including: type of response (online, mail-in, etc.), tenure in a home, presence of non-citizen, state/geography and more [Elliot_2021]_. These are not currently included in our model 
+#. There is some evidence that young children are missed in the post enumeration survey and therefore are missed more than accounted for here [OHare_2019]_ 
+
```

Comment:
Another limitation: we assume that the effects of race and age/sex do not interact and combine additively.